core/src/main/java/feast/core/model/FeatureTable.java

```diff
@@ -107,6 +109,12 @@
   @Column(name = "revision", nullable = false)
   private int revision;
 
+  @Column(name = "is_deleted", nullable = false)
+  private boolean isDeleted;
+
+  @Column(name = "metadata_hash", nullable = false)
```

Comment:
Can we just calculate this hash when needed? why store it in postgres?

Reply:
I was just about to say the same thing.